### ISSUE TITLE
Add stylelint CSS linter

### ISSUE THREAD

**Issue by pdehaan** [mozilla-lockbox/lockbox-extension#352]
It recently caught 2 subtle bugs in a PR, but otherwise our CSS looks solid.

```sh
$ npm i stylelint stylelint-config-standard -D
```

```sh
$ cat .stylelintrc
{
  "extends": "stylelint-config-standard",
  "rules": {
    "color-hex-length": null,
    "comment-empty-line-before": null
  }
}
```

```sh
$ $(npm bin)/stylelint 'src/**/*.css'

src/webextension/firstrun/components/welcome.css
 16:1  ✖  Expected empty line before rule   rule-empty-line-before

src/webextension/list/components/item-summary.css
  7:12  ✖  Expected a leading zero                    number-leading-zero
 13:12  ✖  Expected a leading zero                    number-leading-zero
 21:7   ✖  Expected newline after ","                 selector-list-comma-newline-after
 22:3   ✖  Unexpected unknown property "weight"       property-no-unknown
 30:3   ✖  Unexpected unknown property "font-wight"   property-no-unknown

src/webextension/list/manage/components/homepage.css
 18:21  ✖  Unexpected whitespace at end of line                no-eol-whitespace
 18:22  ✖  Expected newline before "}" of a multi-line block   block-closing-brace-newline-before

src/webextension/popup/unlock/components/unlock-prompt.css
 13:3  ✖  Unexpected empty line before declaration   declaration-empty-line-before

src/webextension/widgets/dialog-box.css
 10:3  ✖  Unexpected longhand value '1em 0 0 0' instead of '1em 0 0'   shorthand-property-no-redundant-values
```

We can also run w/ `--fix` flag, but it only auto-fixes 2 of the 10 issues.

**Comment by devinreams** [comment]
Cool idea, @pdehaan. Thanks for the suggestion.

What do you think @mozilla-lockbox/engineering? Do you agree it's worth linting our CSS, too? I'd personally feel better with it...

**Comment by jimporter** [comment]
Seems good, although I'd probably want to see a few of the rules turned off (e.g. the last one; I almost always use 1-, 2-, or 4-value versions for margin/padding, since I find the 3-value one somewhat confusing).

**Comment by devinreams** [comment]
>Seems good, although I'd probably want to see a few of the rules turned off

Thanks @jimporter.

@pdehaan Did you already have a branch you wanted to publish as a PR? I can also grab this before I continue/finish up work at #331 since I'll want to check my work along the way anyway. 😀

**Comment by jimporter** [comment]
Resolved by #463
